README.md

```diff
@@ -87,6 +87,7 @@ See [example/App.js](example/App.js)
     avatar: 'https://facebook.github.io/react/img/logo_og.png',
   },
   image: 'https://facebook.github.io/react/img/logo_og.png',
+  isSystemMessage: false,
```

Comment:
From my point of view, `fromSystem` would be a better name. `message.fromSystem` seems more readable and less redundant than `message.isSystemMessage`.